static/app/views/dashboards/widgetBuilder/widgetBuilder.tsx

```diff
@@ -1199,7 +1199,7 @@ function WidgetBuilder({
                                           {
                                             from: source,
                                             field: 'title',
-                                            value: state.title ?? '',
```

Comment:
I removed these because Ale confirmed we don't need to know the exact stuff users set. That can be queried in redash if it needs to be.